src/components/FeatureViews/AdminPortal/AdminPortalTabs.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import Tab from '@material-ui/core/Tab';
+import Tabs from '@material-ui/core/Tabs';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+
+const AdminPortalTabs = () => {
+    const [tabIndex, setTabIndex] = useState(0);
+    const handleTabChange = (event, currentTabIndex) => {
+        setTabIndex(currentTabIndex);
+    };
+
+    const useStyles = makeStyles({
+        adminPortalTabsStyle: {
+            marginTop: '2rem',
+        },
+    });
+
+    const classes = useStyles();
+
+    // *** Components for each tab to be created and placed in respective tabContent property in adminPortalTabs array ***
+
+    const adminPortalTabs = [
```

Comment:
Would the tabContent be an actual component to be displayed? 
EX: 
```
 { label: 'overview', tab_id: 0, tabContent: <Overview/>}
```

If that is the way you're thinking of it having a static header is best put as an Object.

Arrays are expensive and looping just to display the component is not fast. (I'm trying to fix the same issue at work lol) 

What i would do is create an object and have it display the component depending the current tab_id. 

We can talk about it later.